KinEcosystem/Core/Environment.swift

```diff
@@ -69,7 +82,7 @@ public enum Environment {
         case .playground:
             return "https://s3.amazonaws.com/assets.kinecosystembeta.com/index.html"
```

Comment:
not sure if this is the right playground webURL?

Reply:
well.

Reply:
Changed